packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -178,13 +179,21 @@ class RenderableTiledMap {
           final src = ts.computeDrawRect(t).toRect();
           final flips = SimpleFlips.fromFlips(tile.flips);
           final size = destTileSize;
+          final scale = size.x / src.width;
+          final anchor = Vector2(src.width / 2, src.height / 2);
+          final offset = Vector2((tx + .5) * size.x, (ty + .5) * size.y)
```

Comment:
Are there cases when `_renderLayer` will run each tick?
If that is the case you'd want to have two private `Vector2` objects outside of this method that you can re-use with `setFrom` or `setValues`, so that you don't have to create new `Vector2` objects here each tick.

Reply:
`_renderLayer` shouldn't be called per tick unless there are some excessive calls to `refreshCache`. Perhaps a scenario with lots of layer visibility changes? But either way a large map would produce a lot of `Vector2` object during load. There isn't really a need for me to store these calculations in a `Vector2` instead of just a pair of doubles so I think I'll just switch to that.